localstack/services/s3/s3_utils.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+S3_STATIC_WEBSITE_HOST_REGEX = r'^(.+).s3-website.localhost.localstack.cloud(:[\d]{0,6})?$'
```

Comment:
nit: Could we move `S3_HOSTNAME_PATTERN` from `s3_listener.py` here as well, to keep the constants in a single place. Also, same comment as for `S3_HOSTNAME_PATTERN` regarding escaping dots in the regex.